src/main/java/lottogame/domain/lotto/Lottos.java

```diff
@@ -0,0 +1,37 @@
+package lottogame.domain.lotto;
+
+import lottogame.domain.Money;
+import lottogame.domain.stats.LottoResult;
+import lottogame.domain.stats.LottoResults;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Lottos {
+    private final List<Lotto> lottos = new ArrayList<>();
+    private LottoResults lottoResults;
+
+    public Lottos(List<Lotto> values) {
+        lottos.addAll(values);
```

Comment:
```suggestion
      this.lottos = new ArrayList<>(values);
```
받은 인자를 바로 초기화해줘도 좋지 않을까요~?

Reply:
이 부분도 리팩토링을 재수정하면서 유의할 수 있도록 했습니다!!!

```java
public class LottoResults {
    ...

    public Map<Rank, Integer> values() {
        return Collections.unmodifiableMap(result);
    }
```

```java
public class LottoNumber {
    private static final int LOTTO_MIN = 1;
    private static final int LOTTO_MAX = 45;
    private static final int LOTTO_NUMBER_VOLUME = 6;
    private final List<Integer> numbers = new ArrayList<>();

    List<Integer> values() {
        return Collections.unmodifiableList(numbers);
    }

   ....
```